src/Symfony/Component/EventDispatcher/EventDispatcherInterface.php

```diff
@@ -25,10 +25,11 @@ interface EventDispatcherInterface extends ContractsEventDispatcherInterface
     /**
      * Adds an event listener that listens on the specified events.
      *
-     * @param int $priority The higher this value, the earlier an event
-     *                      listener will be triggered in the chain (defaults to 0)
+     * @param callable $listener The listener
```

Comment:
The type _actually_  accepted by our implementations is `callable|array{\Closure, string}`. Since the array with the service closure is a somewhat internal functionality, I'm unsure if we want to document it.

Reply:
what about compiling a compatibile closure instead?

Reply:
I don't think the DI component can do that. Apart from that: sure, that would be the nicer solution, but that wouldn't qualify as a bugfix, I guess. 🙂

Reply:
then im in favor of documenting it, so we dont make this mistake again :+1: